src/jvm/main/org/jetbrains/kotlinx/lincheck/Reporter.kt

```diff
@@ -397,10 +409,17 @@ internal fun StringBuilder.appendExecutionScenarioWithResults(
     return this
 }
 
+private fun <T> List<T>.fillWithNullsToSize(requiredSize: Int): List<T?> {
```

Comment:
Let's get rid of this function (see my comments about `Result?` in the `ExecutionResult` class).